ang/civicase/ContactCaseTabCaseList.html

```diff
@@ -0,0 +1,29 @@
+<div>
+  <civicase-case-card mode="long" case="case" ng-repeat="case in caseObj.cases"></civicase-case-card>
+  <div 
+    class="panel panel-default panel-strip-primary civicase__case-card civicase__case-card--contact-record" 
+    ng-repeat="placeholder in loadingPlaceholders"
+    ng-if="!caseObj.isLoaded">
+    <div class="panel-body">
+      <div class="civicase__case-card__row civicase__case-card__row--primary clearfix">
+        <div class="pull-left civicase__case-card-title">
+          <div class="civicase__loading-placeholder__oneline" style="width: 16em; font-size: 18px;"></div>
+        </div>
+        <div class="pull-right">
+          <div class="civicase__loading-placeholder__oneline" style="width: 12em; font-size: 18px;"></div>
+        </div>      
+      </div>
+    </div>
+  </div>
+  <div class="text-center civicase__contact-cases-tab-list-footer">
```

Comment:
You have changed the scss but this is not updated. `civicase__contact-case-tab__case-list__footer`